### ISSUE TITLE
Creating an Ad

### ISSUE THREAD

**Issue by lupuspl** [moxi9/phpfox#429]
When user wants to pay for an ad it is possible via PayPal but when he wants to pay with Activity Points 
nothing happend after clicking "purchase with activity points"
And user has  information before purchasing: "You have 1,215 activity point(s). This will cost 25 activity point(s)."  , so there is enough activity points.

**Comment by one44** [comment]
Is activity points working in V4

**Comment by muhammadnaeemahsan** [comment]
Purchasing with activity points working properly in v4.0.7.
But not show user activity points on there account/ profile or home page.
Not sure how to see user there activity points

**Comment by AnthonyKing** [comment]
check the user group settings / check the block of the user

**Comment by Data66PF** [comment]
Tested with 4.0.8. I made a placement and charged just $1 for the ad. I enabled purchase with points and set the conversion at 1. The user group settings are enabled for purchase with points, for purchasing ads and to see ads. I don't have ad blockers enabled on my browser.

userb creates a front end ad and is given the option to purchase with points. Userb clicks to purchase with points and has enough points to purchase this.

Upon clicking to purchase with points, the ad is paid for.

Admin user goes into admincp >> settings >> ad Campaigns and approves the ad.

The ad is now seen front end.

Tested with win 10 on firefox latest, using neutron default theme with no edits.
